fetchmail_attach_from_folder/views/fetchmail_server.xml

```diff
@@ -61,50 +61,67 @@
                                 />
                             </header>
                             <group colspan="4" col="2">
+                                <!-- Fetch Options -->
                                 <group>
+                                    <separator string="Fetch Options" />
+                                    <field name="active" />
                                     <field name="path" placeholder="INBOX.subfolder1" />
-                                    <field name="model_id" />
-                                    <field name="action_id" />
-                                    <field name="match_algorithm" />
+                                    <field name="fetch_unseen_only" />
+                                    <field name="fetch_last_day_only" />
                                 </group>
-                                <group
-                                    name="group_email_match"
-                                    attrs="{'invisible':
-                                        [('match_algorithm','=','odoo_standard')]}"
-                                >
-                                    <field
-                                        name="model_field"
-                                        placeholder="email"
-                                        attrs="{'required':
-                                            [('match_algorithm','in',['email_exact','email_domain'])]}"
-                                    />
-                                    <field
-                                        name="mail_field"
-                                        placeholder="to,from"
-                                        attrs="{'required':
-                                            [('match_algorithm','in',['email_exact','email_domain'])]}"
-                                    />
-                                    <field name="match_first" />
-                                    <field
-                                        name="domain"
-                                        placeholder="[('state', '=', 'open')]"
-                                    />
-                                    <field
-                                        name="model_order"
-                                        placeholder="name asc"
-                                        attrs="{'readonly':
-                                            [('match_first','==',False)],
-                                            'required':
-                                            [('match_first','==',True)]}"
-                                    />
-                                    <field name="flag_nonmatching" />
+
+                                <!-- Matching Options -->
+                                <group>
+                                    <separator string="Matching Options" />
+                                    <group colspan="2">
+                                      <field
```

Comment:
The field element should be indented 4 spaces under group, not sure why pre-commit prettify did not pick this up.